controllers/numaresourcesscheduler_controller_test.go

```diff
@@ -158,5 +158,52 @@ var _ = ginkgo.Describe("Test NUMAResourcesScheduler Reconcile", func() {
 			gomega.Expect(ok).To(gomega.BeTrue())
 			gomega.Expect(val).ToNot(gomega.BeEmpty())
 		})
+
+		ginkgo.It("should react to owned objects changes", func() {
+			key := client.ObjectKeyFromObject(nrs)
+			_, err := reconciler.Reconcile(context.TODO(), reconcile.Request{NamespacedName: key})
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+			key = client.ObjectKey{
+				Name:      "topo-aware-scheduler-config",
+				Namespace: testNamespace,
+			}
+
+			cm := &corev1.ConfigMap{}
+			gomega.Expect(reconciler.Client.Get(context.TODO(), key, cm)).ToNot(gomega.HaveOccurred())
+
+			key.Name = "secondary-scheduler"
+			dp := &appsv1.Deployment{}
+			gomega.Expect(reconciler.Client.Get(context.TODO(), key, dp)).ToNot(gomega.HaveOccurred())
+
+			initialCM := cm.DeepCopy()
+			cm.Data["somekey"] = "somevalue"
+
+			initialDP := dp.DeepCopy()
+			dp.Spec.Template.Spec.Hostname = "newname"
+			c := corev1.Container{Name: "newcontainer"}
+			dp.Spec.Template.Spec.Containers = append(dp.Spec.Template.Spec.Containers, c)
+
+			err = reconciler.Client.Update(context.TODO(), cm)
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+			err = reconciler.Client.Update(context.TODO(), dp)
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+			key = client.ObjectKeyFromObject(nrs)
+			_, err = reconciler.Reconcile(context.TODO(), reconcile.Request{NamespacedName: key})
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+			key = client.ObjectKeyFromObject(cm)
+			err = reconciler.Client.Get(context.TODO(), key, cm)
```

Comment:
probably same as above - I'm not sure why we need to retry a `Get` though

Reply:
I'm not sure it's necessary, there are plenty of examples in the code of ```Get``` but none of them is in ```Eventually``` loop except ones that are expected to be transitioned.